build.sh

```diff
@@ -735,21 +706,6 @@ case "$COMMAND" in
         exit 0
         ;;
 
-    "verify-osx-swift")
```

Comment:
All these verify- targets which just forward to the test- target are now covered by a `verify-*` target.